xmonad.cabal

```diff
@@ -27,8 +27,7 @@ author:             Spencer Janssen, Don Stewart, Adam Vogt, David Roundy, Jason
                     Ondřej Súkup, Paul Hebble, Shachaf Ben-Kiki, Siim Põder, Tim McIver,
                     Trevor Elliott, Wouter Swierstra, Conrad Irwin, Tim Thelion
 maintainer:         xmonad@haskell.org
-tested-with:        GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.3, GHC == 8.6.5,
-                    GHC == 8.8.4, GHC == 8.10.2
+tested-with:        GHC == 8.4.3, GHC == 8.6.5, GHC == 8.8.4, GHC == 8.10.2
```

Comment:
8.4.4 here as well, perhaps?

Reply:
Ah good point, I suppose we should only have version there that are in the CI (I built it with 8.4.3 just now, so technically it's tested with that, but if the CI builds against 8.4.4 there's the possibility of this becoming outdated of course)

Reply:
It's unlikely to matter, but the comment below says "Keep this in sync" so it'd be nice if it was kept in sync at least initially until someone inevitably forgets :-D

Reply:
Hah, very good point :) 

I'll reduce the version to only the ones in the CI then; `tested-with` doesn't mean "doesn't work with newer versions", after all